agent.Dockerfile

```diff
@@ -8,15 +8,18 @@ COPY go.sum  go.sum
 
 RUN go mod download
 
-COPY cmd/    cmd/
 COPY pkg/    pkg/
+COPY cmd/    cmd/
 
 # Build
+USER root
```

Comment:
I wonder we need root user?

Reply:
without root the build fails with permission error. We follow this pattern in modelmesh/odh-model-controller as well